pkg/autodetect/operatorconfig.go

```diff
@@ -112,3 +127,36 @@ func (c *operatorConfigurationWrapper) GetESPIntegration() ESOperatorIntegration
 func (c *operatorConfigurationWrapper) IsESOperatorIntegrationEnabled() bool {
 	return c.GetESPIntegration() == ESOperatorIntegrationYes
 }
+
+func (c *operatorConfigurationWrapper) SetKafkaIntegration(e interface{}) {
+	var integration string
+	switch v := e.(type) {
+	case string:
+		integration = v
+	case KafkaOperatorIntegration:
+		integration = v.String()
+	default:
+		integration = KafkaOperatorIntegrationNo.String()
+	}
```

Comment:
```suggestion
	integration := KafkaOperatorIntegrationNo.String()
	if v, ok := e.(string); ok {
		integration = v
	} else if v, ok := e.(KafkaOperatorIntegration); ok {
		integration = v.String()
	}
```

Reply:
I would prefer to follow the current approach. Is less "hacky" and more explicit.